internal/ui/operations/details/details.go

```diff
@@ -130,7 +130,23 @@ func (s *Operation) internalUpdate(msg tea.Msg) (*Operation, tea.Cmd) {
 				[]string{"Are you sure you want to split the selected files?"},
 				confirmation.WithStylePrefix("revisions"),
 				confirmation.WithOption("Yes",
-					tea.Batch(s.context.RunInteractiveCommand(jj.Split(s.revision.GetChangeId(), selectedFiles), common.Refresh), common.Close),
+					tea.Batch(s.context.RunInteractiveCommand(jj.Split(s.revision.GetChangeId(), selectedFiles, false), common.Refresh), common.Close),
+					key.NewBinding(key.WithKeys("y"), key.WithHelp("y", "yes"))),
+				confirmation.WithOption("No",
+					confirmation.Close,
+					key.NewBinding(key.WithKeys("n", "esc"), key.WithHelp("n/esc", "no"))),
+			)
+			s.confirmation = model
+			return s, s.confirmation.Init()
+		case key.Matches(msg, s.keyMap.Details.SplitParallel):
```

Comment:
Instead of duplicating this here, you can do something like:

```go
case key.Matches(msg, s.keyMap.Details.Split, s.keyMap.Details.SplitParallel):
isParallel := case key.Matches(msg, s.keyMap.Details.SplitParallel)
```
and pass `isParallel` down to `jj.Split`.